src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -18,18 +18,18 @@ A new web platform API called [Portals](https://github.com/WICG/portals) can hel
+Portals are new HTML elements that can help you keep your front-end simple while allowing seamless navigations with custom transitions. Like dynamic navigations in Single Page Applications with an [App Shell model](https://developers.google.com/web/fundamentals/architecture/app-shell),
 Portals let you dynamically change content with transitions while keeping your Multiple Page Application architecture. But the best part about Portals is they are quite easy to work with.
 
-Think of them like an iframe in that they allow for embedding, but unlike an iframe they also come with features to directly navigate to their content. Seeing is believing: please first check out what we showcased at Chrome Dev Summit last year:
+Think of them like an `<iframe>` in that they allow for embedding, but unlike an `<iframe>` they also come with features to directly navigate to their content. Seeing is believing: please first check out what we showcased at Chrome Dev Summit last year:
 
 <div style="width:100%; padding-top: 56.25%; position: relative;">
   <iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://www.youtube.com/embed/Ai4aZ9Jbsys?start=1081" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
-With classic navigations, users have to wait with a blank screen until the browser finishes rendering the navigation destination. With Portals, users get to experience an animation, while the portal pre-renders content and creates a seamless navigation experience. For example, when the user selects a new chapter in the demo above, the pre-rendered chapter fully animates and covers the screen, while the URL switches to the next chapter.
+With classic navigations, users have to wait with a blank screen until the browser finishes rendering the navigation destination. With Portals, users get to experience an animation, while the `<portal>` pre-renders content and creates a seamless navigation experience. For example, when the user selects a new chapter in the demo above, the pre-rendered chapter fully animates and covers the screen, while the URL switches to the next chapter.
 
-Before Portals, we could have rendered another page using an iframe. We could also have added animations to move the frame around the page. But iframes won’t let you navigate into their content. Portals close this gap, enabling interesting use cases.
+Before Portals, we could have rendered another page using an `<iframe>`. We could also have added animations to move the frame around the page. But an `<iframe>` won’t let you navigate into its content. Portals close this gap, enabling interesting use cases.
 
 ## Try out Portals in Chrome Canary
 
```

Comment:
Code comment should be:

"You can also use the <portal> tag instead."

Reply:
Done.